handle.go

```diff
@@ -8,14 +8,29 @@ import (
 	"time"
 )
 
+type urlpath string
+
+const urlpathDebug urlpath = "/debug"
```

Comment:
Move it to `const` block below (line 18) ?

Reply:
To me it makes sense to group these constants by type because they serve as an enum here.